core/domain/stats_jobs_one_off.py

```diff
@@ -1046,6 +1046,26 @@ def reduce(key, stringified_values):
         yield '%s: %d' % (key, len(stringified_values))
 
 
+class RefreshInteractionRegistryJob(jobs.BaseMapReduceJobManager):
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    # pylint: disable=unused-argument
+    @staticmethod
+    def map(item):
+        yield ('key', 'value')
+
+    @staticmethod
+    def reduce(key, stringified_values):
+        # pylint: disable=protected-access
+        interaction_registry.Registry._refresh()
```

Comment:
Ah, yes an admin button makes more sense (for some reason I didn't think of that at the time...) Nevertheless, this will only be run 2 more times (once for a test, and once for the real migration). Adding the additional button in the admin panel involves quite a bit more than just keeping this job here. It is heavyweight, but it's not too bad and it runs quickly.